extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java

```diff
@@ -466,7 +467,7 @@ void start(@Observes @Priority(Interceptor.Priority.PLATFORM_BEFORE) StartupEven
      *
      * @param event ignored
      */
-    void destroy(@Observes @BeforeDestroyed(ApplicationScoped.class) Object event) {
+    void destroy(@Observes(notifyObserver = Reception.IF_EXISTS) @BeforeDestroyed(ApplicationScoped.class) Object event) {
```

Comment:
styling: static import would be appreciate the code and make it leaner and therefor more explizit, by hiding whats not mandatory to tell and exist.

Reply:
thanks for the mr, hopefully we get 3.0.0 fully released soon 👍